tests/unit/conftest.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
+
+import unittest
+import unittest.mock
+
+import ops.pebble
+import ops.testing
+import pytest
+
+from charm import WordpressCharm
+from tests.unit.wordpress_mock import WordpressPatch
+
+
+@pytest.fixture(scope="function")
+def patch():
+    """Enable WordPress patch system, used in combine with :class:`ops.testing.Harness`.
+
+    Yields:
+        The instance of :class:`tests.unit.wordpress_mock.WordpressPatch`, which can be used to
+        inspect the WordPress mocking system (mocking db, mocking file system, etc).
+    """
+    patch = WordpressPatch()
+    patch.start()
+    yield patch
+    patch.stop()
+
+
+@pytest.fixture(scope="function")
+def harness(patch):
+    """Enable ops test framework harness."""
+    harness = ops.testing.Harness(WordpressCharm)
+    yield harness
+    harness.cleanup()
+
+
+@pytest.fixture(scope="function")
+def app_name():
+    """The name of the charm application."""
+    return "wordpress-k8s"
+
+
+@pytest.fixture(scope="function")
+def setup_replica_consensus(harness, app_name):
```

Comment:
For this fixture and others, it can be helpful to type these (e.g., `def setup_replica_consensus(harness: ops.testing.Harness, app_name: str):`), this will provide auto-complete in some editors and get mypy to check for potential bugs. In fixtures and testing, usually return values are pretty pointless since pytest does magic with the return values so I usually omit them

Reply:
Added, thanks!